chaos/create_actor_failures.go

```diff
@@ -0,0 +1,23 @@
+package chaos
+
+import (
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/specs-actors/actors/builtin"
+	"github.com/filecoin-project/specs-actors/actors/runtime"
+	"github.com/filecoin-project/specs-actors/actors/util/adt"
+	"github.com/ipfs/go-cid"
+)
+
+func (a Actor) CreateAccountActorWithAddr(rt runtime.Runtime, addr *address.Address) *adt.EmptyValue {
+	rt.CreateActor(builtin.AccountActorCodeID, *addr)
+
+	return nil
+}
+
+func (a Actor) CreateUnknownActor(rt runtime.Runtime, addr *address.Address) *adt.EmptyValue {
+	var UnknownActorCodeID cid.Cid
+
+	rt.CreateActor(UnknownActorCodeID, *addr)
```

Comment:
Ideally we want to pass an `addr` here under which we could create an actor (i.e. empty actor as in https://github.com/filecoin-project/specs-actors/blob/master/actors/builtin/init/init_actor.go#L60-L77)